api_docs/construct-narrow.md

```diff
@@ -84,15 +88,20 @@ filters did.
 
 ### Message IDs
 
-The `near` and `id` operators, documented in the help center, use message
-IDs for their operands.
+The `with`, `near` and `id` operators, documented in the help center,
+use message IDs for their operands.
```

Comment:
Is it true that `with` is documented in the help center? I think it isn't, in this version.

Which is probably appropriate, since our plan is to not put it in the search box.

Probably this paragraph can just remain as it is, and `with` can be documented in its own paragraphs separate from the paragraphs about `near` and `id`.

Reply:
updated!